rllib/connectors/tests/test_connector.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from ray.rllib.connectors.connector import Connector, ConnectorPipeline
+
+
+class TestConnectorPipeline(unittest.TestCase):
+    class Tom(Connector):
+        def to_config():
+            return "tom"
+
+    class Bob(Connector):
+        def to_config():
+            return "bob"
+
+    class Mary(Connector):
+        def to_config():
+            return "mary"
+
+    class MockConnectorPipeline(ConnectorPipeline):
+        def __init__(self, ctx, connectors):
+            # Real connector pipelines should keep a list of
+            # Connectors.
+            # Use strings here for simple unit tests.
+            self.connectors = connectors
+
+    def test_sanity_check(self):
+        ctx = {}
+
+        m = self.MockConnectorPipeline(ctx, [self.Tom(ctx), self.Bob(ctx)])
+        m.insert_before("Bob", self.Mary(ctx))
```

Comment:
Where does the Pipeline magically know this string ("Bob") from?

Reply:
I am thinking name here must match the name of the connector.
for example ClipActionConnector is the class. and you say I want to insert_before("ClipActionConnector", AnotherConnector()).
we basically compare the string name with connector_instance.__class__.__name__
let me know if you think it's not a good idea. I feel like add a connector before after another specific connector would be a common use case.